### DISCUSSION TITLE
Have filtes be remembered

### DISCUSSION THREAD

**Discussion by Br1ghtw1ng** [Heroes-Profile/heroesprofile#1068] [Ideas]
Just joined the Patreon, i thought this site was dead, so happy to see it back
It would be great to be able to:
have the filters i chose remembered so the data loads faster, like friend vs. foe, when flipping around during session, and even better if a close my browser, cookies are clear, and i come back, maybe part of my profile?

**Comment by Zemill** [comment]
We currently have some settings that are set on your profile.  Are you look for more?  I assume so since you mention the Friend/Foe page
